sdks/python/setup.py

```diff
@@ -325,7 +325,9 @@ def get_portability_package_data():
               'google-cloud-videointelligence>=2.0,<3',
               'google-cloud-vision>=2,<4',
               'google-cloud-recommendations-ai>=0.1.0,<0.11.0',
-              'google-cloud-aiplatform>=1.26.0, < 2.0'
+              'google-cloud-aiplatform>=1.26.0, < 2.0',
+              # GCP Packages required by YAML
```

Comment:
Do we not need a js2py dependency added as well? (Not to this section, but it's where I can comment.)